datahub_sap_hana/ingestion.py

```diff
@@ -178,3 +200,115 @@ def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
                 wu = item.as_workunit()
                 self.report.report_workunit(wu)
                 yield wu
+
+    def get_column_lineage_view_definitions(self) -> Iterable[Tuple[str, str]]:
+
+        url = self.config.get_sql_alchemy_url()
+        engine = create_engine(url, **self.config.options)
+
+        with engine.connect() as conn:
+            inspector = inspect(conn)
+            schema = inspector.get_schema_names()  # returns a list
+            views = inspector.get_view_names(
+                schema[0])  # returns a list
+
+            for view in views:
+                definition = inspector.get_view_definition(view, schema[0])
+                yield (view, definition)
```

Comment:
Is `view` the name? You can make the code a little easier to read by changing the variable name here
```suggestion
            for view_name in views:
                definition = inspector.get_view_definition(view_name, schema[0])
                yield (view_name, definition)
```